ddp-workspace/projects/ddp-pancan/src/app/components/splash-pages/lms-page/lms-page.component.ts

```diff
@@ -11,19 +17,41 @@ import { FormControl } from '@angular/forms';
 export class LmsPageComponent implements OnInit {
     readonly phone: string;
     readonly email: string;
-    emailToNotify: FormControl;
     readonly AppRoutes = AppRoutes;
 
-    constructor(@Inject('toolkit.toolkitConfig') config: ToolkitConfigurationService) {
+    constructor(
+        private dialog: MatDialog,
+        @Inject('toolkit.toolkitConfig') private config: ToolkitConfigurationService,
+        private translateService: TranslateService,
+        private log: LoggingService
+    ) {
         this.phone = config.lmsPagePhone;
         this.email = config.lmsPageEmail;
     }
 
+    notify(): void {
```

Comment:
do we want to rename to openJoinMailingList?

Reply:
Renamed